tools/vhidcfg-tui

```diff
@@ -150,11 +154,13 @@ vhid_add()
 			${DIALOG_OK})
 				if [ "${mtag}" = "COMMIT" ]; then
 					[ -z "${carp_vhid}" ] && f_dialog_msgbox "vhid must be filled" && continue
-					[ -z "${carp_advskew}" ] && f_dialog_msgbox "vhid must be filled" && continue
-					[ -z "${interface}" ] && f_dialog_msgbox "vhid must be filled" && continue
+					[ -z "${carp_advskew}" ] && f_dialog_msgbox "advskew must be filled" && continue
+					[ -z "${interface}" ] && f_dialog_msgbox "interface must be filled" && continue
+					[ -z "${carp_peer}" ] && f_dialog_msgbox "peer must be filled" && continue
+					[ -z "${carp_peer6}" ] && f_dialog_msgbox "peer6 must be filled" && continue
 					[ -z "${carp_state}" ] && f_dialog_msgbox "state must be filled" && continue
 					[ -z "${carp_pass}" ] && f_dialog_msgbox "vhid must be filled" && continue
-					cbsdsqlrw local "INSERT INTO carp ( vhid,advskew,pass,interface,state ) VALUES ( \"${carp_vhid}\",\"${carp_advskew}\",\"${carp_pass}\", \"${interface}\", \"${carp_state}\" )"
+					cbsdsqlrw local "INSERT INTO carp ( vhid,advskew,pass,interface,peer,peer6,state ) VALUES ( \"${carp_vhid}\",\"${carp_advskew}\",\"${carp_pass}\", \"${interface}\", \"${carp_peer}\", \"${carp_peer6}\", \"${carp_state}\" )"
```

Comment:
FYI : SQLite3 prohibits to use DQS https://www.sqlite.org/quirks.html#dblquote. in CBSD it still works because we enable DQS/quotas when connecting to SQLite3, but it's better to start moving away from double quotes ( '${carp_vhid}' ...